src/components/Deploy/2-ConfigurationStep/Fields/OperationsField/OperationField.tsx

```diff
@@ -0,0 +1,41 @@
+import { classNames } from 'src/utils'
+import { useDeployContext } from 'src/contexts'
+import { DeployMethodsEnum } from 'src/types/deployTypes'
+import { useFocus } from './hooks'
+import { MainInput } from './MainInput'
+import { Operation } from './Operation'
+import { FieldHeader } from 'src/components/Deploy/FieldHeader'
+
+export function OperationsField() {
+  const { mainRef, setSecondaryRef, setFocus } = useFocus()
+  const {
+    state: { operations, selectedDeployMode: deployMethod },
+  } = useDeployContext()
+
+  const isFieldDisabled = deployMethod === DeployMethodsEnum.ALL
```

Comment:
```suggestion
  const isFieldDisabled = deployMethod === DeployMethodsEnum.ALL
```

I don't understand the purpose of isFieldDisabled since you're not showing it in all deployment ?

Reply:
```suggestion
```

You're right, it is not used anymore.

Reply:
Hum, I'll keep it to refactor the step. Maybe I'll merge the ones that are using the DAG.